resources/analysisTools/copyNumberEstimationWorkflow/phasing.sh

```diff
@@ -40,14 +40,15 @@ $PYTHON_BINARY "$TOOL_BEAGLE_CREATE_FAKE_SAMPLES" \
     --out_file "$UNPHASED_TWOSAMPLES" \
     || dieWith "Non zero exit status while creating 2nd sample in vcf-file in phasing.sh"
 
-$JAVA_BINARY \
+$JAVA_BINARY -Xmx9g \
```

Comment:
could you please put this (or at least '9g') to the config so it will become easily adustable from outside? Same please for nthreads

Reply:
This is updated. But not tested yet. My job queues are currently full.

Reply:
thank you for updating this. could you please test it before pulling?

Reply:
Tested, we could merge this now. @vinjana